core/server/worker/src/test/java/alluxio/worker/block/UfsInputStreamManagerTest.java

```diff
@@ -147,8 +147,9 @@ public void testConcurrency() throws Exception {
         threads.add(new Thread(runnable));
       }
       ConcurrencyUtils.assertConcurrent(threads, 30);
-      // Each subsequent check out per thread should be a seek operation
-      Mockito.verify(mSeekableInStreams[0], Mockito.times(numCheckOutPerThread - 1))
+      // Each subsequent check out per thread should be a seek operation, check that at least half
+      // of them call seek method
+      Mockito.verify(mSeekableInStreams[0], Mockito.atLeast(numCheckOutPerThread / 2))
```

Comment:
Sum the seek calls and assert that is equal to the expected amount?

Reply:
The challenge is that the expected number is not a constant but be larger than a certain value, due to jenkins execution stability.

Reply:
`mSeekableInStreams[0]`'s seeks are not constant, but the sum of all the streams should be constant?

Reply:
oh, I see what you mean. That's also viable.